### PR TITLE
Updated to include separate discussion info for partners and open edx

### PR DESCRIPTION
**Pull request by lamagnifica**
Also updates text and an image for DOC-2062 (section download no longer
supported), and removes an out of date and redundant image

@lou-wang, @catong, @aleffert these are pretty lightweight updates to the Learner's guides. Delivery requested by 17 Aug.

### CODE REVIEW

**Comment by aleffert** [comment]
Those arrow icons should be right aligned. That seems like a bug (and probably not an image we want in our docs)

**Comment by aleffert** [comment]
Are we ready to replace that list image? It's not released for the edx.org version of the app.

**Comment by aleffert** [comment]
Otherwise, :+1:

**Comment by lamagnifica** [comment]
yeah, I assumed that the image had some problems, and tried to align the icons myself -- clearly not well enough.
you're also right that I didn't consider that there are different images needed (if we include them) for partners and for open edX. Let me pnder.
Thanks, @aleffert

**Comment by lamagnifica** [comment]
@aleffert I addressed one issue by adding the original image back, and making the images audience-specific. For the other, since we might be delivering doc before sw, I don't think it's crucial to have a final image right now, so noted that we'll probably need a replacement.

**Comment by aleffert** [comment]
To be clear, about the right alignment I meant something more like this 
mob_downloadicon_openedx

**Comment by lamagnifica** [comment]
@aleffert yes, understood. Would rather not spend a while lot of time tweaking images by hand though.

File: en_us/shared/students/SFD_mobile.rst [hunk 1]
```diff
@@ -90,10 +90,24 @@ Can I take a course entirely on my mobile device?
...
 
-Right now, you cannot use the app to participate in course discussions. To
-read or contribute to the discussions, use a web browser on a computer.
+.. The following paragraph describes the features of the edX mobile app for partners/edx.org
+.. Alison, DOC-1840, June 2015
+
+.. only:: Partners
 
-.. Reviewers, I will provide the Open_edX version of this paragraph ^^ in a different PR - Alison
+  Right now, you cannot use the edX mobile apps to participate in course
```
Comment (catong) [inline en_us/shared/students/SFD_mobile.rst:99]: "app" singular?

File: en_us/shared/students/SFD_mobile.rst [hunk 2]
```diff
@@ -90,10 +90,24 @@ Can I take a course entirely on my mobile device?
...
+  discussions. To read or contribute to the discussions, use a web browser on
+  a computer.
+
+.. The following paragraph describes the features of the edX mobile app for Open edX (adds notifications, assessments, discussions)
+.. Alison, DOC-1840, June 2015
+
+.. only:: Open_edX
+
+  You can read and contribute to course discussions in the edX mobile apps
```
Comment (catong) [inline en_us/shared/students/SFD_mobile.rst:108]: "apps" -> "app"?

File: en_us/shared/students/SFD_mobile.rst [hunk 3]
```diff
@@ -131,31 +162,18 @@ if you intend to watch or download a lot of videos. The edX mobile app has a
...
-
-========================================================
-What are the circles to the left of the video name?
-========================================================
-
-The circle indicates whether you have already played a video. A blue circle
-means that you have not started playing the video. A half-filled circle means
-that you played part of the video. A gray circle means that you played the
-entire video.
+The number to the right of a subsection name is the number of videos in that
```
Comment (catong) [inline en_us/shared/students/SFD_mobile.rst]: "is the number" -> "indicates the number of videos"

File: en_us/shared/students/SFD_mobile.rst [hunk 4]
```diff
@@ -131,31 +162,18 @@ if you intend to watch or download a lot of videos. The edX mobile app has a
...
+subsection.
 
 ========================================
 How can I make the video full screen?
 ========================================
 
 Rotate your phone until the video changes to landscape, or select "full
```
Comment (catong) [inline en_us/shared/students/SFD_mobile.rst]: "landscape" -> "landscape mode"

File: en_us/shared/students/SFD_mobile.rst [hunk 5]
```diff
@@ -169,8 +187,7 @@ You can change video speeds on the iPhone app. While a video plays, select
 **Settings** (the "gear" icon) at the lower right, and then select **Video
 Speed**.
 
-On the Android app, videos currently run only in their original speeds. EdX is
-working on the ability to change video speeds.
+On the Android app, videos currently run only in their original speeds.
```
Comment (catong) [inline en_us/shared/students/SFD_mobile.rst]: "run" is a bit odd. "play" instead? and "at" a speed, not "in" a speed.

"videos currently play only at their original speed."

File: en_us/shared/students/SFD_mobile.rst [hunk 6]
```diff
@@ -210,6 +227,16 @@ How do I delete downloaded videos?
 
 #. Select **Delete**.
 
+========================================================
+What are the circles to the left of the video name?
```
Comment (catong) [inline en_us/shared/students/SFD_mobile.rst]: Is it one circle per video? In that case "What is the circle to the left of the video name?" If you want to keep things plural, "What are the circles to the right of video names?"

**Comment by catong** [comment]
:+1:  with a couple of minor edits.

File: en_us/shared/students/SFD_mobile.rst (hunk 1 above)
Comment (lamagnifica) [inline en_us/shared/students/SFD_mobile.rst:99]: Turns out that referring to our "app" in the singular is a bit of a misnomer. There are two apps, one that runs on Android smartphones and one that runs on iPhones. They are completely different, though the engineering and UX teams have worked really hard to make the user interfaces as similar as possible. It's a bit of a struggle to get every reference in the doc correct though.

File: en_us/shared/students/SFD_mobile.rst (hunk 5 above)
Comment (lamagnifica) [inline en_us/shared/students/SFD_mobile.rst]: thank you for making suggestions on the original text -- I agree with all of them, though I have not been prioritizing rewriting existing content. I will try to make as many of them as possible.

**Pull request merged**